src/core/iomgr/tcp_posix.c

```diff
@@ -199,11 +199,11 @@ static void tcp_continue_read(grpc_exec_ctx *exec_ctx, grpc_tcp *tcp) {
   msg.msg_controllen = 0;
   msg.msg_flags = 0;
 
-  GRPC_TIMER_BEGIN(GRPC_PTAG_RECVMSG, 0);
+  GPR_TIMER_BEGIN("recvmsg", 1);
```

Comment:
I changed the meaning... getting rid of id and making this 'important' -- it turns out to be useful to have important start/end scopes.